### ISSUE TITLE
[Feature Request] DirectX Implementation

### ISSUE THREAD

**Issue by sekyi-joshua** [asfdfdfd/renpy-live2d#3]
Hello, there.

Is it possible for you to implement DirectX support?
I'm not sure how often DirectX is used in Renpy, though.
If you don't think it is important, then perhaps it is better to just leave it.

**Comment by asfdfdfd** [comment]
Yes. It is possible and i think that DirectX support is important feature. But as for now it is not top priority for me. I leave task opened because someday i (or somebody interested in DirectX) will implement it.

**Comment by marcussacana** [comment]
Isn't better use vulkan? Will works with linux too, and with the MoltenVK project the osx too.

**Comment by asfdfdfd** [comment]
I do not write all rendering code for Live2D myself. 99% of it is already done in CubismNativeFramework by Live2D. I write thin wrapper to integrate rendering of CubismNativeFramework with RenPy. 

So, to use vulkan both RenPy and CubismNativeFramework should support it.
